test/test_torch.py

```diff
@@ -938,6 +939,240 @@ def test_add(self):
 
         # [res] torch.add([res,] tensor1, value, tensor2)
 
+    def mb_rand(self, *dims):
```

Comment:
All those tests should go into `test_jit`, because they have nothing to do with the core package. You can create a new test case for them.